config/application.rb

```diff
@@ -216,8 +220,22 @@ class Application < Rails::Application
     config.session_store :active_record_store, :secure => !!SETTINGS[:require_ssl]
   end
 
-  def self.setup_console
-    ENV['IRBRC'] = File.expand_path('../irbrc', __FILE__)
-    puts "For some operations a user must be set, try User.current = User.first"
+  class << self
+    def setup_console
+      ENV['IRBRC'] = File.expand_path('../irbrc', __FILE__)
+      puts "For some operations a user must be set, try User.current = User.first"
+    end
+
+    def dynflow
+      return @dynflow if @dynflow.present?
+      if defined?(ForemanTasks)
+        @dynflow ||= ForemanTasks.dynflow
+      else
+        @dynflow ||= Foreman::Dynflow.new
+        @dynflow.require!
+        @dynflow.initialize!
+        @dynflow
+      end
```

Comment:
I don't think that's a good idea, that code calls initialize! every time (which creates a new Dynflow world)